### ISSUE TITLE
The 'by' step clause causes an error

### ISSUE THREAD

**Issue by SSaulenas** [BalticAmadeus/AblFormatter#159]
Example:

```progress
define variable i as integer no-undo.

do i = 5 to 10 by 2:
    message "HELLO".
end.
```

![image](https://github.com/user-attachments/assets/9ac2f42a-a8e8-4260-84c8-51889082b715)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/82

**Comment by eglekaz** [comment]
Fixed with 0.0.42 version of tree-sitter-abl
